pkg/utils/log/log.go

```diff
@@ -68,30 +67,30 @@ func setLogger(v *int, debug *bool) {
 		encoderConf := zap.NewDevelopmentEncoderConfig()
 		encoderConf.EncodeLevel = zapcore.CapitalColorLevelEncoder
 		encoder = zapcore.NewConsoleEncoder(encoderConf)
-
-		opts = append(opts, zap.Development(), zap.AddStacktrace(zap.ErrorLevel))
 	} else {
 		encoderConf := zap.NewProductionEncoderConfig()
 		encoderConf.MessageKey = "message"
 		encoderConf.TimeKey = "@timestamp"
 		encoderConf.EncodeTime = zapcore.ISO8601TimeEncoder
 		encoder = zapcore.NewJSONEncoder(encoderConf)
 
-		opts = append(opts, zap.AddStacktrace(zap.ErrorLevel))
 		if zapLevel.Level() > zap.DebugLevel {
 			opts = append(opts, zap.WrapCore(func(core zapcore.Core) zapcore.Core {
 				return zapcore.NewSampler(core, time.Second, 100, 100)
 			}))
 		}
 	}
 
-	sink := zapcore.AddSync(os.Stderr)
-	opts = append(opts, zap.AddCallerSkip(1), zap.ErrorOutput(sink))
-	log := zap.New(zapcore.NewCore(&crzap.KubeAwareEncoder{Encoder: encoder, Verbose: dev.Enabled}, sink, zapLevel))
-	log = log.WithOptions(opts...)
-	log = log.With(zap.String("ver", getVersionString()))
-
-	crlog.SetLogger(zapr.NewLogger(log))
+	opts = append(opts, zap.Fields(zap.String("ver", getVersionString())))
```

Comment:
Since `opts` is only used here now, it would be better to remove the var declaration above and just write something like:

```go
opts := []zap.Option{zap.Fields(zap.String("ver", getVersionString()))}
```

Reply:
done